app/pages/nominees/WhatIsTheNameOfOrganisationPage.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nominees
+
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object WhatIsTheNameOfOrganisationPage  extends QuestionPage[String] {
+
+  override def path: JsPath = JsPath \ toString
+
+  override def toString: String = "nameOfOrganisation"
```

Comment:
for the sake of having all the nominee details nicely organised, could you  change the path to something like `JsPath \ "nomineeOrganisation" \ toString`?

Reply:
override def path: JsPath =  NomineeId.path \ toString
 override lazy val toString: String = "organisationName"
would be more useful

Reply:
welp - too late, let's change that up in the next ticket then